cdap-api/src/main/java/co/cask/cdap/api/artifact/PluginConfigurer.java

```diff
@@ -25,7 +26,7 @@
  * This interface provides methods to register plugin usage in a CDAP Program. The registered plugins will be
  * available at the runtime of the Program.
  */
-public interface PluginConfigurer {
+public interface PluginConfigurer extends DatasetConfigurer {
```

Comment:
is DatasetConfigurer ever used on its own? If not maybe its better to have just one interface?